platform/features/conductor-redux/src/README.md

```diff
@@ -0,0 +1,187 @@
+## Notes
+API is notional for now, based on use-cases identified below. Possible the 
+use cases are not sufficient, so please include in comments 
+any other use cases you'd like to see. 
+
+Plan now is to start building out test suite for the use cases identified below
+in order to get the API functional. Need to discuss how UI aspects of timeline will be implemented.
+Propose in place refactoring of existing timeline rather than starting again.
+
+Some caveats / open questions
+* I don't understand the use case shown on page 52 of UI sketches. It shows RT/FT, with deltas, 
+with inner interval unlocked. Not sure what result would be, has inner end switched to fixed?
+Also example on page 55 in real-time where inner end < now. Is there a use case for this? Semantically, it's saying
+show me real time, but stale data. Why would a user want this? My feeling is that if the inner 
+OR outer ends are moved behind NOW in real-time mode then you drop into historical mode.
```

Comment:
We discussed this a bit on Friday, and were in overall agreement that there wasn't a use case for end < now with RT. Some alternatives were discussed (I think your proposal to drop into historical mode was among them) but no final decision made. In any event, don't think this has any impact on the API